### PR TITLE
Add possibility to prioritize production by area

### PR DESCRIPTION
**Pull request by pnuu**
This PR makes it possible to prioritize areas to be processed in certain order by giving them priority value. If priorities are not given, all defined areas are processed together. All areas with the same priority are processed together.

### CODE REVIEW

File: trollflow2/launcher.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+def message_to_jobs(msg, product_list):
+    jobs = OrderedDict()
+    priorities = get_area_priorities(product_list)
+    # TODO: check the uri is accessible from the current host.
+    input_filenames = [urlparse(uri).path for uri in gen_dict_extract(msg.data, 'uri')]
+    for prio, areas in priorities.items():
+        jobs[prio] = OrderedDict()
+        jobs[prio]['input_filenames'] = input_filenames.copy()
+        jobs[prio]['input_mda'] = msg.data.copy()
+        jobs[prio]['product_list'] = {}
```
Comment (mraspaud) [inline trollflow2/launcher.py]: Shouldn't we add the `common` section too ?

Comment (pnuu) [reply trollflow2/launcher.py]: The `common` section is inside the `product_list` on the same level as `products`.

Comment (mraspaud) [reply trollflow2/launcher.py]: I don't see it, where does this happen ?

Comment (mraspaud) [reply trollflow2/launcher.py]: ah, sorry, got it

**Pull request merged**